test/Orchard.Tests/Tokens.Content/SlugServiceTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Orchard.Tokens.Content.Services;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Orchard.Tests.Tokens.Content
+{
+    public class SlugServiceTests
+    {
+        private SlugService _slugService;
+
+        public SlugServiceTests()
+        {
+            _slugService = new SlugService();
+        }
+
+        [Fact]
+        public void ShouldStripContiguousDashes()
+        {
+            var slug = _slugService.Slugify("a - b");
+            Assert.Equal("a-b", slug);
+        }
+
+        [Fact]
+        public void ShouldStripContiguousDashes2()
```

Comment:
Use a [Theory] when you need to create multiple test cases for the same scenario.
Here is an example:
https://github.com/sebastienros/jint/blob/458698df5c325e8c2c9bc611820740c7f6398bb1/Jint.Tests/Runtime/EngineTests.cs#L57